app/controllers/api/v1/topic_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Api
+  module V1
+    class TopicController < ApplicationController
+      before_action :authenticate_api_v1_user!
+
+      def index
+        render json: Topic.all
```

Comment:
Maybe you should use the view here, sending  @topics variable, like @topics  = Topic.all